src/scores/continuous.py

```diff
@@ -41,11 +41,8 @@ def mse(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
     """
 
     error = fcst - obs
-
     squared = error * error
-
-    if weights is not None:  # pragma: no-cover
-        raise NotImplementedError("Weights handling not implemented, placeholder for API spec")  # pragma: no-cover
+    squared = scores.functions.apply_weights(squared, weights)
```

Comment:
Just confirming that `rmse` will work correctly with the weighting applied before the `sqrt`? 
@nicholasloveday